src/pages/members.js

```diff
@@ -1,12 +1,66 @@
-import React from "react"
+import React, { Component } from "react"
 import Layout from "../components/layout"
 import SEO from "../components/seo"
+import { fetchResults } from "../utils/api";
 
-const MembersPage = () => (
-  <Layout>
-    <SEO title=" Chapter Members" />
-    <h1>Chapter Members</h1>
-  </Layout>
-)
+class MembersPage extends Component {
+  state = {
+    fetchedMembersList: [],
+    hasError: false
+  }
+
+  async componentDidMount() {
+    try {
+      const response = await fetchResults("http://my-json-server.typicode.com/WomenWhoCodeCincy/wwcode-cincy-database/members");
+      if (response.success) {
+        this.setState({ fetchedMembersList: response.results, hasError: false });
+      } else {
+        this.setState({ hasError: true });
+      }
+    } catch {
+      this.setState({ hasError: true });
+    }
+  }
+
+  // Backup Image in case url from database is broken
+  addDefaultSrc(ev) {
+    ev.target.src = "https://avatars3.githubusercontent.com/u/13358086?s=200&v=4"
+  }
```

Comment:
Nice!

Reply:
Thanks! That was fun to figure out how to do